src/views/data/colours.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
```

Comment:
I see what you mean about the lack of format agnostic token storage. I'll see if I can find any token utility tool that could potentially help on this front

Reply:
Yeah that would be incredibly useful